src/wazuh_db/wdb_parser.c

```diff
@@ -6031,3 +6041,80 @@ int wdb_parse_task_delete_old(wdb_t* wdb, const cJSON *parameters, char* output)
 
     return result;
 }
+
+// 'agents' DB command parsing
+
+int wdb_parse_vuln_cve(wdb_t* wdb, char* input, char* output) {
+    int result = OS_INVALID;
+    char * next;
+    const char delim[] = " ";
+    char *savedptr = NULL;
+
+    next = strtok_r(input, delim, &savedptr);
+
+    if (!next){
+        snprintf(output, OS_MAXSTR + 1, "err Missing vuln_cve action");
+    }
+    else if (strcmp(next, "insert") == 0) {
+        next = strtok_r(NULL, delim, &savedptr);
+        result = wdb_parse_agents_insert_vuln_cve(wdb, next, output);
+    }
+    else if (strcmp(next, "clear") == 0) {
+        result = wdb_parse_agents_clear_vuln_cve(wdb, output);
+    }
+    else {
+        snprintf(output, OS_MAXSTR + 1, "err Invalid vuln_cve action: %s", next);
+    }
+
+    return result;
+}
+
+
+int wdb_parse_agents_insert_vuln_cve(wdb_t* wdb, char* input, char* output) {
+    cJSON *data = NULL;
+    const char *error = NULL;
+    int ret = OS_INVALID;
+
+    data = cJSON_ParseWithOpts(input, &error, TRUE);
+    if (!data) {
+        mdebug1("Global DB Invalid JSON syntax when inserting agent.");
+        mdebug2("Global DB JSON error near: %s", error);
+        snprintf(output, OS_MAXSTR + 1, "err Invalid JSON syntax, near '%.32s'", input);
+    }
+    else {
+        cJSON* j_name = cJSON_GetObjectItem(data, "name");
+        cJSON* j_version = cJSON_GetObjectItem(data, "version");
+        cJSON* j_architecture = cJSON_GetObjectItem(data, "architecture");
+        cJSON* j_cve = cJSON_GetObjectItem(data, "cve");
+        // Required fields
+        if (!cJSON_IsString(j_name) || !cJSON_IsString(j_version) || !cJSON_IsString(j_architecture) ||!cJSON_IsString(j_cve)) {
+            mdebug1("Global DB Invalid JSON data when inserting vuln_cve. Not compliant with constraints defined in the database.");
+            snprintf(output, OS_MAXSTR + 1, "err Invalid JSON data, near '%.32s'", input);
+        }
+        else {
+            ret = wdb_agents_insert_vuln_cve(wdb, j_name->valuestring, j_version->valuestring, j_architecture->valuestring, j_cve->valuestring);
+            if (OS_SUCCESS != ret) {
+                mdebug1("Global DB Cannot execute SQL query; err database %s/%s.db: %s", WDB2_DIR, WDB_GLOB_NAME, sqlite3_errmsg(wdb->db));
+                snprintf(output, OS_MAXSTR + 1, "err Cannot execute Global database query; %s", sqlite3_errmsg(wdb->db));
```

Comment:
The debug messages reference `Global DB`

Reply:
Fixed